app/styles/_state.scss

```diff
@@ -48,6 +48,10 @@
   }
 }
 
+.working-error {
+  background: $info-background-color !important;
```

Comment:
@MynahMarie is there any way to make this work without an `!important` ? 

see above on line 41 `&.visible {` can we add a similar block such as `&.working-error {`

Reply:
Sure I will give it a try! I used `!important` to override the background property of the `.error` class but I will see if I manage to use `&.visible` instead.